converter/src/test/java/gov/cms/qpp/acceptance/cpc/FixtureErrorData.java

```diff
@@ -0,0 +1,44 @@
+package gov.cms.qpp.acceptance.cpc;
+
+import gov.cms.qpp.conversion.model.error.ErrorCode;
+
+import java.util.List;
+
+public class FixtureErrorData {
```

Comment:
Another case where just a bit of documentation will be helpful in understanding the data this class is used to represent.